### PR TITLE
ARACHNE-2419. Added datanode view edit screen

### PR DESCRIPTION
**Pull request by johnSamilin**

### CODE REVIEW

File: src/modules/DataCatalog/components/DataNode/ViewEdit/components/EditCommonData/container.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+  * Unless required by applicable law or agreed to in writing, software
+  * distributed under the License is distributed on an "AS IS" BASIS,
+  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  * See the License for the specific language governing permissions and
+  * limitations under the License.
+  *
+  * Company: Odysseus Data Services, Inc.
+  * Product Owner/Architecture: Gregory Klebanov
+  * Authors: Pavel Grafkin, Alexander Saltykov, Vitaly Koulakov, Anton Gackovka, Alexandr Ryabokon, Mikhail Mironov
+  * Created: Tuesday, June 26, 2018 2:06 PM
```
Comment (pavgra) [inline src/modules/DataCatalog/components/DataNode/ViewEdit/components/EditCommonData/container.js]: We have the standard format. Pls, follow it

File: src/modules/DataCatalog/components/DataNode/ViewEdit/components/EditDataNodeTitle/container.js [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+    return {
+      ...ownProps,
+      ...stateProps,
+      ...dispatchProps,      
+      async doSubmit({ name }) {
+        const result = await dispatchProps.update(
+          { id: stateProps.id },
+          {
+            name,
+            description: stateProps.description,
```
Comment (pavgra) [inline src/modules/DataCatalog/components/DataNode/ViewEdit/components/EditDataNodeTitle/container.js]: Why are descr and org fields in EditDataNode_Title_ popup? They should not be here

File: src/modules/DataCatalog/components/DataNode/ViewEdit/components/EditCommonData/style.scss [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  * Product Owner/Architecture: Gregory Klebanov
+  * Authors: Pavel Grafkin, Alexander Saltykov, Vitaly Koulakov, Anton Gackovka, Alexandr Ryabokon, Mikhail Mironov
+  * Created: Tuesday, June 26, 2018 2:06 PM
+  *
+  */
+
+@import 'styles/vars-and-mixins.scss';
+ 
+.#{$namespace} { 
+  &edit-common-data {
```
Comment (agackovka) [inline src/modules/DataCatalog/components/DataNode/ViewEdit/components/EditCommonData/style.scss]: tab

File: src/modules/DataCatalog/components/DataNode/ViewEdit/components/EditDataNodeTitle/style.scss [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+  * Authors: Pavel Grafkin, Alexander Saltykov, Vitaly Koulakov, Anton Gackovka, Alexandr Ryabokon, Mikhail Mironov
+  * Created: Monday, June 25, 2018 2:13 PM
+  *
+  */
+
+@import 'styles/vars-and-mixins.scss';
+ 
+.#{$namespace} { 
+  &edit-data-node-title {
+
```
Comment (agackovka) [inline src/modules/DataCatalog/components/DataNode/ViewEdit/components/EditDataNodeTitle/style.scss]: empty

File: src/modules/DataCatalog/components/DataNode/ViewEdit/components/EditDataNodeTitle/presenter.jsx [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+  * Created: Monday, June 25, 2018 2:13 PM
+  *
+  */
+
+import React from 'react';
+import {
+  Modal,
+  Form,
+  FormInput,
+  } from 'arachne-ui-components';
```
Comment (agackovka) [inline src/modules/DataCatalog/components/DataNode/ViewEdit/components/EditDataNodeTitle/presenter.jsx]: indent

File: src/modules/DataCatalog/components/DataNode/ViewEdit/container.js [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+import { DataNodeSelectors } from './selectors';
+import { ModalUtils } from 'arachne-ui-components';
+import presenter from './presenter';
+import { modal, form } from 'modules/DataCatalog/const';
+
+const selectors = (new DataNodeSelectors).build();
+
+/** @augments { Component<any, any> } */
+export class ViewEdit extends Component {
+  static get propTypes() {
```
Comment (agackovka) [inline src/modules/DataCatalog/components/DataNode/ViewEdit/container.js]: can be deleted

File: src/modules/DataCatalog/components/DataNode/ViewEdit/components/EditCommonData/container.js [hunk 7]
```diff
@@ -0,0 +1,107 @@
...
+  *
+  * Unless required by applicable law or agreed to in writing, software
+  * distributed under the License is distributed on an "AS IS" BASIS,
+  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  * See the License for the specific language governing permissions and
+  * limitations under the License.
+  *
+  * Company: Odysseus Data Services, Inc.
+  * Product Owner/Architecture: Gregory Klebanov
+  * Authors: Pavel Grafkin, Alexander Saltykov, Vitaly Koulakov, Anton Gackovka, Alexandr Ryabokon, Mikhail Mironov
```
Comment (agackovka) [inline src/modules/DataCatalog/components/DataNode/ViewEdit/components/EditCommonData/container.js]: what was done by Mikhail Mironov here?)

**Pull request merged**